### PR TITLE
Allow action bar to be hidden on Plotly tables

### PR DESCRIPTION
**Pull request by dhgrajeda**
This option allows hiding the action bar on Plotly tables. This allows tables Notebook Views to look a little more clean when the bar is not needed.

**Normal table layout**
hide-toolbar-hidden

**Table layout with actionbar hidden**
hide-toolbar-showing

### CODE REVIEW

File: src/sql/workbench/contrib/query/browser/gridPanel.ts [hunk 1]
```diff
@@ -736,7 +742,14 @@ export abstract class GridTableBase<T> extends Disposable implements IView {
 	private rebuildActionBar() {
 		let actions = this.getCurrentActions();
 		this.actionBar.clear();
-		this.actionBar.push(actions, { icon: true, label: false });
+		if (this.options.displayActions) {
+			this.actionBar.push(actions, { icon: true, label: false });
+		}
+	}
+
+	public showActionBar(display: boolean) {
```
Comment (alanrenmsft) [inline src/sql/workbench/contrib/query/browser/gridPanel.ts:750]: I think I left the same comment in the other PR you have.

1. let's use getter/setter, showActionBar(boolean) seems weird to me.
2. the name should be consistent, lets use showActionBar in the options too.
3. only rebuild the action bar when the value is actually changed.

```TypeScript
public get showActionBar():boolean {
    return this.options.showActionBar;
}

public set showActionBar(v: boolean) {
    if(this.options.showActionBar !== v) {
        this.options.showActionBar = v;
        this.rebuildActionBar();
    }
}
```

**Comment by alanrenmsft** [comment]
it is always good to add a screenshot in the PR description.

**Review by alanrenmsft — APPROVED** [review]

**Comment by Charles-Gagnon** [comment]
@dhgrajeda Is this ready for review? It's currently still in the draft state

**Comment by dhgrajeda** [comment]
> @dhgrajeda Is this ready for review? It's currently still in the draft state

@Charles-Gagnon Yes, just switched it to ready for review

**Pull request merged**